packages/core/__tests__/language-server/requests.test.ts

```diff
@@ -0,0 +1,328 @@
+import { Project } from 'glint-monorepo-test-utils';
+import { describe, beforeEach, afterEach, test, expect } from 'vitest';
+import { stripIndent } from 'common-tags';
+import * as ts from 'typescript';
+
+describe('Language Server: Requests', () => {
```

Comment:
Can we call this something like `Language Server: Organize Imports`/`organize-imports.test.ts`?

Technically lots of messages that come in are requests; it's just that most of the ones we have implemented today have dedicated handlers in the `vscode-languageserver` API like `connection.onDefinition(...)` rather than using the generic `connection.onRequest(...)`.